mne/viz/_3d.py

```diff
@@ -1061,12 +1044,8 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
                              for pick in seeg_picks])
 
     # initialize figure
-    if fig is None:
-        fig = _mlab_figure(bgcolor=(0.5, 0.5, 0.5), size=(800, 800))
-    if interaction == 'terrain' and fig.scene is not None:
-        fig.scene.interactor.interactor_style = \
-            tvtk.InteractorStyleTerrain()
-    _toggle_mlab_render(fig, False)
+    renderer.init((800, 800), (0.5, 0.5, 0.5))
```

Comment:
please keep keywords names explicit.

Reply:
I tried to find a generic name for a function making basic pre-configuration of the visualization.